src/utils/utils.ts

```diff
@@ -36,8 +36,38 @@ export const imageURL = (
 
 export const getNumItems = async (uuid: string) => {
   const apiUrl = `https://api.repo.nypl.org/api/v2/collections/${uuid}/items`;
-  const apiKey = process.env.AUTH_TOKEN;
+  const res = await apiCall(apiUrl);
+  return res.numItems || 0;
+};
+
+/**
+ * Calls Repo API collectins/uuid/items endpoint
+ * @param {string} uuid - collection uuid
+ */
 
+export const getItemsFromUUID = async (uuid: string) => {
+  const apiUrl = `https://api.repo.nypl.org/api/v2/collections/${uuid}/items`;
+  return apiCall(apiUrl);
+};
+
+/**
+ * Returns the uuid, API uri, and numResults of an item given an identifier type and identifier value.
+ * @param {string} identifierType - the identifier type
+ * @param {string} identifier - the identifier value
+ */
+
+export const getAPIUri = async (identifierType: string, identifier: string) => {
+  const apiUrl = `https://api.repo.nypl.org/api/v2/items/${identifierType}/${identifier}`;
```

Comment:
It doesn't have to be for this PR but this should be environment-aware and the domain should be an env var. When it's on QA it should be the QA API domain and for prod it should use this domain. If not for this PR, add a todo so it doesn't get lost.